src/Enable.Extensions.Queuing.RabbitMQ/Internal/RabbitMQQueueClient.cs

```diff
@@ -6,14 +6,15 @@
 using Enable.Extensions.Queuing.Abstractions;
 using RabbitMQ.Client;
 using RabbitMQ.Client.Events;
+using RabbitMQ.Client.Exceptions;
 
 namespace Enable.Extensions.Queuing.RabbitMQ.Internal
 {
     internal class RabbitMQQueueClient : BaseQueueClient
     {
         private const string RedeliveryCountHeaderName = "x-redelivered-count";
 
-        private readonly ConnectionFactory _connectionFactory;
+        private readonly RabbitMQConnectionManager _connectionFactory;
```

Comment:
This variable is called `_connectionFactory`, but it's type is `RabbitMQConnectionManager`. Can we use "factory" consistently, rather than using both "manager" and "factory" terms, i.e. can we rename `RabbitMQConnectionManager` to `RabbitMQConnectionFactory`?